prime-dotnet-webapi/Migrations/ApiDbContextModelSnapshot.cs

```diff
@@ -11682,8 +11689,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Navigation("IdentificationDocuments");
 
+                    b.Navigation("LinkedEnrolment");
+
                     b.Navigation("OboSites");
 
+                    b.Navigation("PaperEnrolment");
+
```

Comment:
Need to re-generate the migration after changing the navigation property names

Reply:
> 

Nice catch! Re-generated.